api/src/main/java/org/apache/brooklyn/api/effector/ParameterType.java

```diff
@@ -45,4 +47,6 @@
      * @return The default value for this parameter, if not supplied during an effector call.
      */
     public T getDefaultValue();
+
+    public boolean isSensitive();
```

Comment:
I really worry about adding this - do we actually respect it properly? In many places we just have a `ConfigBag` without the `ParameterType` definition, so we can only infer from the name whether it looks like a password/credential.

If we include `isSensitive` here, then it will set a false expectation of what we'll do with it.

Should we just rely on `sanitize`, and get rid of `isSensitive`?